### ISSUE TITLE
A mamba awesome list 

### ISSUE THREAD

**Issue by pengzhangzhi** [state-spaces/mamba#132]
Hi @tridao and @albertfgu , I really appreciate your work! As a community, I curate an awesome list of mamba papers https://github.com/pengzhangzhi/Awesome-Mamba. I am wondering if you can post it on your readme? I can help with that if you are interested!

**Comment by albertfgu** [comment]
Really cool, glad you're interested! One of my students actually has started a comprehensive list of related resources here: https://github.com/AvivBick/awesome-ssm-ml

I will publicize it at some point relatively soon, but a bit swamped. Maybe you could contribute there if you have any new resources or other suggestions?

**Comment by pengzhangzhi** [comment]
Yeah! Thanks for bringing this up! To differentiate from the awesome-SSM, I might want to focus on Mamba.
